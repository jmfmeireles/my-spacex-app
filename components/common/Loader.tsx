import { ReactElement } from 'react';

import styles from '../../styles/Loader.module.css';

const Loader = () => {
  return (
    <div className={styles.wrapper}>
      <svg viewBox="-24 -24 48 48" className={styles.svg}>
        <circle
          cx="0"
          cy="0"
          r="20"
          fill="none"
          strokeWidth="4"
          className={styles.circle}
        ></circle>
      </svg>
    </div>
  );
};

export default Loader;
