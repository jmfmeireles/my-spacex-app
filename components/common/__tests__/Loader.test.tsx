import { render } from '@testing-library/react';
import Loader from '../Loader';

it('renders Loader properly', () => {
  const { container } = render(<Loader />);
  expect(container).toMatchSnapshot();
});
