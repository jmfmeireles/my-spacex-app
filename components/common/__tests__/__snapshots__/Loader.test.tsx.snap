// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loader properly 1`] = `
<div>
  <div
    class="wrapper"
  >
    <svg
      class="svg"
      viewBox="-24 -24 48 48"
    >
      <circle
        class="circle"
        cx="0"
        cy="0"
        fill="none"
        r="20"
        stroke-width="4"
      />
    </svg>
  </div>
</div>
`;
